atlasdb-commons/src/test/java/com/palantir/common/compression/StreamCompressionTests.java

```diff
@@ -78,52 +70,60 @@ public void testCanDecompressGzipAsLz4() throws IOException {
                 .isEqualTo(data);
     }
 
-    @Test
-    public void testEmptyStream() throws IOException {
-        initializeStreams(new byte[0]);
+    @ParameterizedTest(name = "{index} {0} compression")
+    @MethodSource("getParameters")
```

Comment:
Commenting here, but applied to all the classes so far: try to name these method sources closer to what they are. In this case we could have something like `@MethodSource("streamCompressionAlgorithms")` which is way more descriptive of what we're passing than just saying `"getParameters"`